.eslintrc.js

```diff
@@ -69,13 +69,21 @@ module.exports = {
             patterns: ['@/features/*/*'],
           },
         ],
-        'linebreak-style': ['error', 'unix'],
+        'linebreak-style': ['error', 'windows'],
```

Comment:
why you add windows? what if somebody with Linux OS or Mac OS will try work with our project, it will be preferred to be unix, not windows, so return back to UNIX and our files must be in LF, not in CRLF https://stackoverflow.com/questions/1552749/difference-between-cr-lf-lf-and-cr-line-break-types